src/DssSpell.t.sol

```diff
@@ -646,40 +646,90 @@ contract DssSpellTest is DSTest, DSMath {
         assertEq(sumlines, values.vat_Line);
     }
 
-//    function testFailWrongDay() public {
-//        vote();
-//        scheduleWaitAndCastFailDay();
-//    }
-
-//    function testFailTooEarly() public {
-//        vote();
-//        scheduleWaitAndCastFailEarly();
-//    }
-
-//    function testFailTooLate() public {
-//        vote();
-//        scheduleWaitAndCastFailLate();
-//    }
-
-    function testSpellIsCast() public {
-        string memory description = new DssSpell().description();
-        assertTrue(bytes(description).length > 0);
-        // DS-Test can't handle strings directly, so cast to a bytes32.
-        assertEq(stringToBytes32(spell.description()),
-                stringToBytes32(description));
-
-        if(address(spell) != address(MAINNET_SPELL)) {
-            assertEq(spell.expiration(), (now + 4 days + 2 hours));
-        } else {
-            assertEq(spell.expiration(), (SPELL_CREATED + 4 days + 2 hours));
-        }
+   function testFailWrongDay() public {
+       vote();
+       scheduleWaitAndCastFailDay();
+   }
+
+   function testFailTooEarly() public {
+       vote();
+       scheduleWaitAndCastFailEarly();
+   }
+
+   function testFailTooLate() public {
+       vote();
+       scheduleWaitAndCastFailLate();
+   }
+
+   function testOnTime() public {
+       vote();
+       scheduleWaitAndCast();
+   }
+
+    // function testSpellIsCast() public {
+    //     string memory description = new DssSpell().description();
+    //     assertTrue(bytes(description).length > 0);
+    //     // DS-Test can't handle strings directly, so cast to a bytes32.
+    //     assertEq(stringToBytes32(spell.description()),
+    //             stringToBytes32(description));
+
+    //     if(address(spell) != address(MAINNET_SPELL)) {
+    //         assertEq(spell.expiration(), (now + 4 days + 2 hours));
+    //     } else {
+    //         assertEq(spell.expiration(), (SPELL_CREATED + 4 days + 2 hours));
+    //     }
+
+    //     vote();
+    //     scheduleWaitAndCast();
+    //     assertTrue(spell.done());
+
+    //     checkSystemValues(afterSpell);
+
+    //     checkCollateralValues(afterSpell);
+    // }
+
+    function test_nextCastTime() public {
+        hevm.warp(1606161600); // Nov 23, 3:00pm (could be cast Nov 26)
 
         vote();
-        scheduleWaitAndCast();
-        assertTrue(spell.done());
+        spell.schedule();
+
+        uint256 monday_10am  = 1606748400; // Nov 30
+
+        /*** Day tests */
```

Comment:
Would you mind adding the following test here:

```
hevm.warp(monday_10am); // Monday, 10am
assertEq(spell.nextCastTime(), monday_10am);
```

It passes, but I just wanted to be sure there wasn't an issue with the `gt` calculation.